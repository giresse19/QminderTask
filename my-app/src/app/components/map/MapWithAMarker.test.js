
import React from "react";

import { withGoogleMap, GoogleMap } from "react-google-maps"
import _ from "lodash";

import {  configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() });

const MapWithAMarker = withGoogleMap(props => (
  <GoogleMap
    ref={props.onMapLoad}
    defaultZoom={12}
    defaultCenter={{ lat: 58.378025, lng: 26.728493 }}
    onClick={props.onMapClick}
  />
));

describe(`GoogleMap`, () => {
  it(`should render`, () => {
    const googleMap = shallow(
      <MapWithAMarker
        containerElement={
          <div style={{ height: `100%`  }} />
        }
        mapElement={
          <div style={{ height: `100%` }} />
        }
        onMapLoad={_.noop}
        onMapClick={_.noop}
      />
    );
    let tree = shallowToJson(googleMap);
    expect(tree).toMatchSnapshot();
  });
});