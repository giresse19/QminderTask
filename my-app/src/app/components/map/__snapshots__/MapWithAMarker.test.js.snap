// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleMap should render 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div
    style={
      Object {
        "height": "100%",
      }
    }
  />
  <div />
</div>
`;
